lineman/app/components/validation_errors/validation_errors.haml

```diff
@@ -0,0 +1,3 @@
+.lmo-validation-error
```

Comment:
This is a nifty little directive which takes a subject ($scope.user) and a field (name), and displays validation errors for that field once we get a 422 back. Wee!